appveyor.yml

```diff
@@ -1,19 +1,11 @@
 version: "{build}"
-os: Windows Server 2012 R2
-branches:
-  only:
-    - master
+platform: x64
+environment:
+  APM_TEST_PACKAGES:
+  matrix:
+  - ATOM_CHANNEL: stable
+  - ATOM_CHANNEL: beta
+build_script:
+  - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/atom/ci/master/build-package.ps1'))
```

Comment:
Same as the above comment, you'll need to switch to the following to use the system Node.js for linting:
```
https://raw.githubusercontent.com/Arcanemagus/ci/atomlinter/build-package.ps1
```